### PR TITLE
Add combineVideos.ts

### PR DESCRIPTION
**Pull request by nh758**
Rewrite combine video based on convert.js
WIP
- Not tested

### CODE REVIEW

**Comment by nh758** [comment]
It's working now and has a test written. Though it includes avi files to test the combination. I'm thinking this might be better to test manually, rather than include it in integration tests?

**Comment by megahirt** [comment]
image

Integration tests passing on Windows

**Comment by megahirt** [comment]
@nh758 do you want to take this PR out of "draft" status and then we can merge it in?  Is it ready to merge?

File: public/cli/lib/commands/combineVideos.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { paths } from '../path-constants';
+import { spawnSync } from 'child_process';
+import winston from 'winston';
+import { EventEmitter } from 'events';
+import { writeFileSync } from 'fs';
+import tempy from 'tempy';
+
+export async function combineVideos(videoPaths: Array<string>, outputFilePath: string, notifyEvent?: EventEmitter) {
```
Comment (irahopkinson) [inline public/cli/lib/commands/combineVideos.ts:8]: Use the simpler notation `string[]` instead of `Array<string>`.

**Review by irahopkinson — CHANGES_REQUESTED** [review]
👍 LGTM with one minor fix up.

File: public/cli/lib/commands/combineVideos.ts (hunk 1 above)
Comment (irahopkinson) [reply public/cli/lib/commands/combineVideos.ts]: Done. Also added return type and fixed typo in comment.

**Review by irahopkinson — APPROVED** [review]

**Pull request merged**